cognite/powerops/resync/models/v2/dms/data_models/views/dayAheadBids-SHOPMultiScenarioMethod.view.yaml

```diff
@@ -0,0 +1,38 @@
+space: '{{dayahead_model_space}}'
+externalId: SHOPMultiScenarioMethod
+name: SHOPMultiScenarioMethod
+filter:
+  equals:
+    property:
+    - node
+    - type
+    value:
+      space: '{{type_space}}'
+      externalId: DayAheadSHOPMultiScenarioMethod
+implements:
+- space: '{{dayahead_model_space}}'
+  externalId: BidMethod
+  version: '{{dayaheadbid_BidMethod_version}}'
+  type: view
+version: '{{dayaheadbid_SHOPMultiScenarioMethod_version}}'
+properties:
+  shopCases:
```

Comment:
Realised that `Case` may cause confusion - in the CogShop data model, a Case is a Scenario (ModelTemplate + mappingsOverride + commands) plus starttime and endtime, but here it is more like a SHOP model (more similar to ModelTemplate in the CogShop model).